h2o-r/h2o-package/R/predict.R

```diff
@@ -0,0 +1,49 @@
+#'
+#' H2O Prediction from R
+#'
+#' Provides the method h2o.predict with which you can predict a MOJO or POJO Jar model
+#' from R.
+#'
+
+
+
+#' @param model  String with file name of MOJO or POJO Jar
+#' @param json  JSON String with inputs to model
+#' @param classpath  (Optional) Extra items for the class path of where to look for Java classes, e.g., h2o-genmodel.jar
+#' @return Returns an object with the prediction result
+#' @importFrom jsonlite fromJSON
+#' @examples
+#' \donttest{
+#' library(h2o)
+#' h2o.predict_json('~/GBM_model_python_1473313897851_6.zip', '{"C7":1}')
```

Comment:
Examples should be reproducible so we can test package with `--run-dontrun --run-donttest`. Best to use some tiny dataset (maybe iris) and very basic model.